packages/melody-compiler/.npmignore

```diff
@@ -0,0 +1,3 @@
+**/__tests__/**
+src
```

Comment:
I am actually in favor of letting `src/` folder stay since we have `jsnext:main` on package json's. also for rare cases that allows you to build packages manually. We have that line in code-frame and the root level historical reasons most likely.

Reply:
Okay. I'll remove `src` but after this https://github.com/trivago/melody/pull/42 `jsnext:main` points to `lib`. But it's good to have src in final package. :)

Reply:
you are right :) I totally forgot about that :-D